README.md

```diff
@@ -4,14 +4,20 @@
 
 ---
 #### Table of Contents
-* [Want to contribute?](#want-to-contribute)
-    * [Requirements](#requirements)
-    * [First-Time Setup](#first-time-setup)
-        * [DB Setup](#db-setup-with-ms-sql)
-        * [Slack Setup](#slack-setup)
-    * [Local Configuration Files](#local-configuration-files)
-* [Running & Debugging (VS Code)](#running--debugging-vs-code)
-* [Helpful Links](#helpful-links)
+- [Egg Party 🥚🎉](#egg-party-%f0%9f%a5%9a%f0%9f%8e%89)
+    - [Egg party is a Slack App for thanking teammates, raising chickens, and having a rowdy ol' time](#egg-party-is-a-slack-app-for-thanking-teammates-raising-chickens-and-having-a-rowdy-ol-time)
+      - [Table of Contents](#table-of-contents)
```

Comment:
Are these guys necessary? I guess I didn't really think of the tagline as a navigable heading

Reply:
Again, I'm pretty sure I left everything else that was already there alone.